src/hooks/pool/useUserYield.ts

```diff
@@ -81,23 +127,32 @@ const formatUserYield = memoize(
 );
 
 export const useUserYields = (
-  marketsData: MarketDataType[]
+  marketsData: MarketDataType[],
+  userAddress?: string
 ): SimplifiedUseQueryResult<UserYield>[] => {
   const poolsFormattedReservesQuery = usePoolsFormattedReserves(marketsData);
   const userSummaryQuery = useUserSummariesAndIncentives(marketsData);
+  const userMeritIncentivesQuery = useUserMeritIncentives(userAddress);
 
   return poolsFormattedReservesQuery.map((elem, index) => {
     const selector = (
       formattedPoolReserves: FormattedReservesAndIncentives[],
       user: FormatUserSummaryAndIncentivesResponse
     ) => {
-      return formatUserYield(formattedPoolReserves, user);
+      // Get merit incentives data separately
+      const meritIncentives = userMeritIncentivesQuery.data;
+      return formatUserYield(
+        formattedPoolReserves,
+        user,
+        meritIncentives,
+        marketsData[index].marketTitle
```

Comment:
```suggestion
        marketsData[index].market
```

Reply:
updated in b762090b6bc2a698b0021aaf3f90b3c876dea271